sound/soc/intel/boards/sof_sdw_maxim.c

```diff
@@ -140,9 +155,17 @@ int sof_sdw_mx8373_init(struct snd_soc_card *card,
 	if (info->amp_num == 2)
 		dai_links->init = spk_init;
 
-	info->codec_card_late_probe = mx8373_sdw_late_probe;
-
-	dai_links->ops = &max_98373_sdw_ops;
+	maxim_part_id = info->part_id;
+	switch (maxim_part_id) {
+	case SOF_SDW_PART_ID_MAX98363:
+		dai_links->ops = &max_98363_sdw_ops;
+		break;
+	default:
```

Comment:
I prefer to add a MAX98373 case and return -EINVAL in the default case.
```
default:
		dev_err(card->dev, "Invalid maxim_part_id %#x\n", maxim_part_id);
		return -EINVAL;
```

Reply:
ack done